packages/apps/hr/src/features/employeeSlice.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {createAsyncThunk, createSlice} from '@reduxjs/toolkit';
+import {
+  generateInifiniteScrollCases,
+  handlerApiCall,
+} from '@axelor/aos-mobile-core';
+import {searchManagedEmployee as _searchManagedEmployee} from '../api/employee-api';
+
+export const searchManagedEmployee = createAsyncThunk(
+  'employee/searchManagedEmployee',
+  async function (data, {getState}) {
+    return handlerApiCall({
+      fetchFunction: _searchManagedEmployee,
+      data,
+      action: 'Hr_SliceAction_SearchManagedEmployee',
+      getState,
+      responseOptions: {isArrayResponse: true},
+    });
+  },
+);
+
+const initialState = {
+  loading: true,
+  moreLoading: false,
+  isListEnd: false,
+  managedEmployeeList: [],
+};
+
+const employeeSlice = createSlice({
+  name: 'employee',
+  initialState,
+  extraReducers: builder => {
+    generateInifiniteScrollCases(builder, searchManagedEmployee, {
+      loading: 'loading',
+      moreLoading: 'moreLoading',
+      isListEnd: 'isListEnd',
+      list: 'managedEmployeeList',
+    });
+  },
+});
```

Comment:
You can not use infinite scroll system here with the api request you created as you can not give a page. You asked for all data at once.
Moreover, the objective here is to get the number of employee managed by the used not really the list, so maybe you can use total on request instead of getting the list. 

1. Modify the api request to set a limit on items to fetch otherwise the request can become really heavy (you can also add page mangement if you want to keep the generateInfiniteScrollCases)
2. Ask for total in handlerApiCall (returnTotal or resturnTotalWithData)
3. Modify slice to match the new format of fetched data